MergerService/src/run.cs

```diff
@@ -138,15 +138,18 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils)
             if (task.Parameters is null || task.Parameters.Batches is null || task.Parameters.Sources is null)
             {
                 this._logger.LogWarning($"Task parameters are invalid. Task: {task.ToString()}");
+                taskUtils.UpdateReject(task.JobId, task.Id, task.Attempts, "Task parameters are invalid", false);
```

Comment:
error here will cause the service to crush

Reply:
Added try catch.